### ISSUE TITLE
Crash with Cpu Isolation

### ISSUE THREAD

**Issue by VGrol** [cjbassi/gotop#52]
```
- Arch Linux 4.17.14
- Fish & Bash
- Termite
- Gotop 1.4.1
```

Behavior:
Isolating CPU cores via Cpuset (cset) results in Gotop crashing on initializing.
This does not happen if Gotop is already running and isolation is achieved afterwards.
Only after isolating cores before running Gotop, does Gotop crash

It throws the following exception
```
~> Gotop
Error caught. Exiting program.

Issue with CPU percentages.

Diagnostics:
self.Count: 2
gopsutil.Counts(): 2
len(percents): 8
percents: [20.000000000004547 0 0 0 16.494845360824865 0 0 0]
self.interval: 1s

panic: 1

goroutine 6 [running]:
github.com/cjbassi/gotop/src/utils.Error(0x5845e0, 0xf, 0xc420202000, 0x83)
	/home/cjbassi/.local/src/github.com/cjbassi/gotop/src/utils/utils.go:49 +0x23f
github.com/cjbassi/gotop/src/widgets.(*CPU).update(0xc4201cada0)
	/home/cjbassi/.local/src/github.com/cjbassi/gotop/src/widgets/cpu.go:57 +0x646
created by github.com/cjbassi/gotop/src/widgets.NewCPU
	/home/cjbassi/.local/src/github.com/cjbassi/gotop/src/widgets/cpu.go:38 +0x2f4
```

To me, at least, this is very low priority, but I stumbled across it while messing around with Cset.
So I figured I'd report it anyways.
I have a feeling any CPU isolation would result in the same problem, since It doesn't like not being able to read/interact as nicely with the isolated cores.

Interestingly enough, Htop doesn't seem to have a problem with reporting the isolated cores. 
Might be worth while checking out how they handle that.

**Comment by joeky888** [comment]
```
Android 8.1 aarch64
Termux
apt-get install golang
gotop 1.5.0
```

Same error.

**Comment by cjbassi** [comment]
@joeky888 Could you clarify what environment you're running gotop with? And are you also using cpu isolation?

**Comment by joeky888** [comment]
> Could you clarify what environment you're running gotop with?

Android app Termux https://play.google.com/store/apps/details?id=com.termux
gotop 1.5
go 1.11.1
apt install golang clang
go get -u github.com/cjbassi/gotop

Crash on startup.

> And are you also using cpu isolation?

Nope.

**Comment by cjbassi** [comment]
@joeky888 Thanks for the info. Even though you're getting the same error, I think these are separate issues and it would be best if you created a new issue for the error you're getting. And also include the details of your OS/platform etc and the exact error gotop is giving you. Thanks!

**Comment by joeky888** [comment]
> if you created a new issue for the error

Ok I'll do it.

Same error happens on arm64 Yocto Linux.

**Comment by cjbassi** [comment]
@VGrol So if I have this right, running gotop on isolated cores should cause it to only utilize those cores, but it should still display all of them in the graph? And you're saying htop displays all the cores?
